### DISCUSSION TITLE
json表单二维数组tabl表格样式问题

### DISCUSSION THREAD

**Discussion by LeeFly-cn** [jqhph/dcat-admin#1855] [General]
table中的icon如何去掉, 表格没有对齐,展示样式不好看,怎么解决呢?
![9f8c5c832f3348e575fc6e1140c7f5b](https://user-images.githubusercontent.com/29645727/180674834-25e3b09e-c3b0-43be-8269-2b93a7513278.jpg)

**Comment by yiming0** [comment]
> table中的icon如何去掉

定义字段时，使用 `prepend()` 方法，参数为空字符串 `''`，例：
```
$form->text('field')->prepend(''); 
````

**Comment by laradocs** [comment]
去掉 icon 不一定能达到你想要的效果:joy:
